Makefile

```diff
@@ -6,9 +6,14 @@ export PATH := $(PATH):$(GOPATH)/bin
 INSTALL := /usr/bin/install
 DBDIR := /var/run/redis/sonic-db/
 GO ?= /usr/local/go/bin/go
+GOROOT ?= $(shell $(GO) env GOROOT)
 TOP_DIR := $(abspath ..)
 MGMT_COMMON_DIR := $(TOP_DIR)/sonic-mgmt-common
 BUILD_DIR := build/bin
+FORMAT_CHECK = $(BUILD_DIR)/.formatcheck
+FORMAT_LOG = $(BUILD_DIR)/go_format.log
+# Find all .go files excluding vendor, build, and patches files
+GO_FILES := $(shell find . -type f -name '*.go' ! -path './vendor/*' ! -path './build/*' ! -path './patches/*' ! -path './proto/*')
```

Comment:
In case of swsscommon, the go file is auto-generated by a foolproof swig tool, which does not require the go formatting. So, can  you add 'swsscommon/*' to the excluded paths?